test/crio_secrets.bats

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+	cleanup_test
+}
+
+IMAGE="redis:alpine"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT ${STORAGE_OPTS}"
+
+MOUNT_PATH_1="$TESTDIR/secrets"
+mkdir ${MOUNT_PATH_1}
```

Comment:
you can wrap all this in a `setup()` function, bats (the test running) will call `setup()` before everything

Reply:
fixed